test/serialization/base64UrlSpecTests.ts

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+import * as assert from "assert";
+import base64UrlSpec from "../../lib/serialization/base64UrlSpec";
+import { deserializeTest, serializeTest } from "./specTest";
+
+describe("base64UrlSpec", () => {
+  it("should have \"Base64Url\" for its typeName property", () => {
+    assert.strictEqual("Base64Url", base64UrlSpec.specType);
+  });
+
+  describe("serialize()", () => {
+    describe("with strict type-checking", () => {
+      function base64UrlSerializeWithStrictTypeCheckingTest(args: { propertyPath?: string[], value: Buffer, expectedResult: string | Error }): void {
+        serializeTest({
```

Comment:
I don't see a definition for this in the diff--is this basically the same suite just migrated based on the new serializer structure?

Reply:
Yeah, basically.